amqp/src/main/scala/akka/stream/contrib/amqp/model.scala

```diff
@@ -4,6 +4,7 @@
 package akka.stream.contrib.amqp
 
 import scala.collection.immutable.Seq
```

Comment:
It adds quite a bit of repetition. What about:
```scala
package object amqp {
  val Seq = scala.collection.immutable.Seq
  type Seq[+A] = scala.collection.immutable.Seq[A]
}
```

This will make sure that whenever Seq is used in our package it is an immutable one.